components/button/button.css

```diff
@@ -2,10 +2,310 @@
+  text-transform: uppercase;
+  transition: background-color 0.25s ease-in;
+  color: var(--primary-color);
+}
+
+.btn-link {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  transition: background-color 0.25s ease-in;
+  border-radius: 4px;
+  color: var(--primary-color);
+}
+
+.btn-link:hover {
+  color: var(--black);
+  background-color: var(--primary-color);
+}
+
+.btn-secondary-container {
+  margin: 2rem;
+  width: 70%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 2px solid var(--black);
+  background-color: var(--primary-color);
+}
+
+.btn-secondary {
+  margin: 2rem;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  position: relative;
+  font-weight: 500;
+  font-size: 2rem;
+  background-color: var(--secondary-color);
+  border: 2px;
+  text-transform: uppercase;
+  border-radius: 4px;
+  transition: background-color 0.2s ease;
+}
+
+.btn-secondary:hover {
+  background-color: var(--primary-color);
+}
+
+.btn-secondary-outline {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  border: 1px solid black;
+  transition: background-color 0.25s ease-in;
+}
+
+.btn-secondary-text {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  transition: background-color 0.25s ease-in;
+  color: var(--secondary-color);
+}
+
+.btn-secondary-link {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  transition: background-color 0.25s ease-in;
+  border-radius: 4px;
+  color: var(--secondary-color);
+}
+
+.btn-secondary-link:hover {
+  color: var(--black);
+  background-color: var(--secondary-color);
+}
+
+.btn-icon-container {
+  margin: 2rem;
+  width: 70%;
+  height: 10rem;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 2px solid var(--black);
+}
+
+.fas {
+  cursor: pointer;
+  font-size: 2rem;
+}
+
+.far {
+  cursor: pointer;
+  font-size: 2rem;
+}
+
+.fas:hover {
+  color: var(--primary-color);
+}
+
+.far:hover {
+  color: var(--primary-color);
+}
+
```

Comment:
fas and far have the same properties. You can mention them together, separated by comma

Reply:
Thanks for pointing